editor/src/components/inspector/fill-hug-fixed-control.tsx

```diff
@@ -85,6 +85,7 @@ export const FillFixedHugControlId = (segment: 'width' | 'height'): string =>
 
 export const FillContainerLabel = 'Fill container' as const
 export const FixedLabel = 'Fixed' as const
+export const ScaledLabel = 'Scaled' as const
```

Comment:
This change is also in https://github.com/concrete-utopia/utopia/pull/4401/files#diff-5a58a0bbc5c3742fd34d2e6e769c411de15fb95986bda7a6baea085451067cddR88
Is that right?